src/restconfig/src/main/java/org/geoserver/rest/security/RestConfigXStreamPersister.java

```diff
@@ -0,0 +1,30 @@
+/* (c) 2019 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.rest.security;
+
+import java.util.logging.Logger;
+import org.geoserver.config.util.XStreamPersister;
+import org.geoserver.config.util.XStreamPersisterInitializer;
+import org.geoserver.rest.security.xml.JaxbUser;
+import org.geotools.util.logging.Logging;
+
+/**
+ * @author ImranR
+ * @since 2.15

```

Comment:
GeoServer only "API" points are considered the interfaces for extension points, no real need to add "@since" markers